test/config_parser_SUITE.erl

```diff
@@ -2281,8 +2286,10 @@ mod_muc_light(_Config) ->
           T(#{<<"rooms_in_rosters">> => true})),
     ?errh(T(#{<<"backend">> => <<"frontend">>})),
     ?errh(T(#{<<"host">> => <<"what is a domain?!">>})),
+    ?errh(T(#{<<"host">> => <<"invalid..com">>})),
     ?errh(T(#{<<"host">> => [<<"invalid.sub@HOST@">>]})),
     ?errh(T(#{<<"host">> => [<<"invalid.sub.@HOST@.as.well">>]})),
+    ?errh(T(#{<<"host">> => [<<"inv@lidsub.@HOST@">>]})),
```

Comment:
This fails because of a list instead of a single host. If you change it, you should get your coverage up to 100% (now it's missing this one case).
```suggestion
    ?errh(T(#{<<"host">> => <<"inv@lidsub.@HOST@">>})),
```